lndr-backend/src/Lndr/EthereumInterface.hs

```diff
@@ -163,15 +163,14 @@ interpretUcacLog change = do
 -- | Verify that a settlement payment was made using a 'txHash' corresponding to
 -- an Ethereum transaction on the blockchain and the associated addresses and
 -- eth settlment amount.
-verifySettlementPayment :: BilateralCreditRecord -> LndrHandler ()
-verifySettlementPayment (BilateralCreditRecord creditRecord _ _ (Just txHash)) = do
+verifySettlementPayment :: BilateralCreditRecord -> Integer -> LndrHandler ()
```

Comment:
Why is this necessary given the field `settlementAmount` on the `BilateralCreditRecord`? I don't think you should add this `Integer` parameter to this function. If for some reason you need to modify the settlementAmount, do so in the `BilateralCreditRecord` passed to this function.